models/ironbank/ethereum/ironbank_ethereum_borrow.sql

```diff
@@ -7,16 +7,17 @@
 ) }}
 
 SELECT
-'ironbank' AS project,
-'1' AS version,
 evt_block_number AS block_number,
 evt_block_time AS block_time,
 evt_tx_hash AS tx_hash,
-evt_index,
+evt_index AS `index`,
 borrower,
-i.underlying_token_address AS asset_address,
-borrowAmount AS borrow_amount
-FROM (
-    SELECT * FROM {{ source('ironbank_ethereum', 'CErc20Delegator_evt_Borrow') }}
-) events
-LEFT JOIN {{ ref('ironbank_ethereum_itokens') }} i ON events.contract_address = i.contract_address
\ No newline at end of file
+itokens.symbol,
+itokens.underlying_symbol,
+itokens.underlying_token_address AS underlying_address,
+borrow.borrowAmount / power(10,itokens.underlying_decimals) AS borrow_amount,
+borrow.borrowAmount / power(10,itokens.underlying_decimals)*p.price AS borrow_usd
+FROM {{ source('ironbank_ethereum', 'CErc20Delegator_evt_Borrow') }} borrow
+LEFT JOIN {{ ref('ironbank_ethereum_itokens') }} itokens ON borrow.contract_address = itokens.contract_address
+LEFT JOIN prices.usd p ON p.minute = date_trunc('minute', borrow.evt_block_time) AND p.contract_address = itokens.underlying_token_address
```

Comment:
can you ensure ```prices.usd``` uses ref() syntax and not hardcode?

Reply:
also, this join relies on another left join -- if the first join returns null, are you comfortable with null prices?

Reply:
this applies to all models (don't want to over-comment)

Reply:
> can you ensure `prices.usd` uses ref() syntax and not hardcode?

How about using source syntax? I found other people used source syntax.

Reply:
> also, this join relies on another left join -- if the first join returns null, are you comfortable with null prices?

Yes.

Reply:
> > can you ensure `prices.usd` uses ref() syntax and not hardcode?
> 
> How about using source syntax? I found other people used source syntax.

yes, sorry, source :)